cdap-proto/src/main/java/io/cdap/cdap/proto/sourcecontrol/UpdateSourceControlMetaRequest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.proto.sourcecontrol;
+
+import io.cdap.cdap.proto.id.ApplicationId;
+import java.util.Objects;
+
+public class UpdateSourceControlMetaRequest {
```

Comment:
in general, the classes in proto represent something in the REST APIs. This should be moved internally to app-fabric.

Reply:
moved to app-fabric. Renamed class to `ApplicationSourceControlMeta`.